src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php

```diff
@@ -72,9 +72,9 @@ protected function configure()
     /**
      * {@inheritdoc}
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $cliOutput)
```

Comment:
Yea, and it makes more sense to me to do the opposite: use `OutputInterface $output` here, then rename the other variable, like `$styleOutput = new SymfonyStyle(...)`. That's mostly because we tend to always call the argument `$output`.